### DISCUSSION TITLE
Where to Enable DLC'

### DISCUSSION THREAD

**Discussion by calimota** [Foxlider/FASTER#223] [Q&A]
DLCs next to Clear Keys is not there anymore to enable all the other dlcs

**Comment by jupster** [comment]
Hey have a look at this, https://github.com/Foxlider/FASTER/pull/194 We relocated them.

**Comment by calimota** [comment]
I only see a quarter of the mods what about the other one that use to be in
the dropped down menu to

**Reply by jupster** [reply]
Make sure you are on the latest build

**Comment by calimota** [comment]
I am, the only ones that pop up are the ones that are shown they use to be
like the other addons on it

**Reply by jupster** [reply]
you might need to post a picture of what you are experiencing

**Reply by jupster** [reply]
![370547548-9154e399-434e-4793-89ed-656267dafc11](https://github.com/user-attachments/assets/fec9b2c3-d462-4cbe-a08e-2765024fb2a6)

**Reply by jupster** [reply]
now with EF 

if you can show me your issue that would be great
